src/config.xml

```diff
@@ -3510,6 +3510,20 @@ where `loc1` and `loc2` can be relative or absolute paths or URLs.
  If the \c EXTERNAL_PAGES tag is set to \c YES, all external pages will be listed
  in the related pages index. If set to \c NO, only the current project's
  pages will be listed.
+]]>
+      </docs>
+    </option>
+  </group>
+  <group name='Msc' docs='Configuration options related to message sequence charts (MSC)'>
+    <option type='string' id='MSCGEN' format='file' defval=''>
+      <docs>
+<![CDATA[
+ You can define MSCs within doxygen comments using the \ref cmdmsc "\\msc"
+ command. Doxygen will either use an internal tool to produce the chart, or if
+ specified the <code>MSCGEN</code> tag allows you to use an external program
+ for that. In the latter case, the external program shall be callable as
```

Comment:
Better to use `\c MSCGEN` for consistency.

Reply:
Done